types/office-js-preview/index.d.ts

```diff
@@ -10035,7 +10035,7 @@ declare namespace Office {
          * - `InvalidAttachmentId`: The attachment identifier does not exist.
          *
          * @param attachmentId - The identifier of the attachment to remove. The maximum string length of the `attachmentId`
-         *                       is 200 characters in Outlook on the web, and 100 characters in Outlook on Windows.
+         *                       is 200 characters in Outlook on the web and Windows.
```

Comment:
Feel free to take or leave this suggestion, but I think it sounds slightly less clunky.
```suggestion
         *                       is 200 characters in Outlook on the web and on Windows.
```
(Here and elsewhere)

Reply:
+1